csharp/ql/lib/semmle/code/csharp/dataflow/internal/ExternalFlow.qll

```diff
@@ -406,6 +405,30 @@ Declaration interpretBaseDeclaration(string namespace, string type, string name,
   )
 }
 
+pragma[inline]
+private Declaration interpretExt(Declaration d, ExtPath ext) {
+  ext = "" and result = d
+  or
+  ext.getToken(0) = "Attribute" and
+  (
+    not exists(ext.getToken(1)) and
+    result.(Attributable).getAnAttribute().getType() = d and
+    not result instanceof Property and
+    not result instanceof Indexer
```

Comment:
Would we be backwards compatible if we removed these two lines?

Reply:
Yes, but I think we should take a conscious decision and break the backwards compatibility - as it was quite surprising that tagging a property meant that it worked for setters.
Also, the public documentation doesn't contain the explanation on, how to use the `ext` field, so it is most likely not heavily used (yet).